css/style.css

```diff
@@ -49,8 +49,34 @@ body {
 .center-text {
   text-align: center !important; }
```

Comment:
Not related to this pull request in particular I think, but it seems you are using important inside your utility classes. After looking into the `main.scss` file I found that you are importing the `components` after `utilities`. A suggestion would be to import `utilities` at the end of file. This would ensure that when using the utility classes, they get applied due to order of precedence (unless some other property overrides it)

Reply:
Hey, fixed the structure and redundant code